Expensify/App
# Use showStates in report params so that report state filters work in OldDot

CC: @yuwenmemon 


### Details


### Fixed Issues

$ https://github.com/Expensify/Expensify/issues/180872

### Tests
For this to work locally, this PR is also needed https://github.com/Expensify/Web-Expensify/pull/32177/files

Also I couldn't get the dang bank set up to work I got stuck on adding an address. So instead I changed this line to just be true:https://github.com/Expensify/App/blob/ab5acf460511f97a9bf2daf6e0eabe60cfd5fe8d/src/pages/workspace/WorkspacePageWithSections.js#L55

In staging we already have accounts with VBA set up.

### QA Steps
1. Get a user on NewDot, with a Workspace and VBA set up on their account.
2. Through workspace settings go to `Pay bills`,  `Reimburse receipts`, and `View Unpaid Invoices` verify that when you click on those, you are redirected to OldDot with the correct report filters applies:
> Pay bills > opens a new browser tab in OldDot on the reports page, filtered by the relevant workspace and report type bills in the processing and approved states.
>Reimburse receipts > Opens a new browser tab and links to the OldDot Reports page filtered by the relevant workspace and expense reports in the Closed state awaiting reimbursement.
>View unpaid invoices > opens a new browser tab in OldDot on the reports page, filtered by the relevant workspace and report type invoices in the processing state awaiting payment.

### Tested On


### Screenshots


#### Web
![image](https://user-images.githubusercontent.com/19364431/137229582-6a7f90c8-8d84-426b-9d75-4ed1199ce1ac.png)
![image](https://user-images.githubusercontent.com/19364431/137229573-8fcc818a-6a8a-42cf-ac58-3de7890b617a.png)

![image](https://user-images.githubusercontent.com/19364431/137229598-091e4572-91a6-4376-83e7-7bd780d35ac8.png)
![image](https://user-images.githubusercontent.com/19364431/137229614-1a9bb818-40d6-4799-a7ed-043b7e8d4809.png)


![image](https://user-images.githubusercontent.com/19364431/137229533-7c7171b5-bd1a-409d-839f-bb9e39ab9856.png)
![image](https://user-images.githubusercontent.com/19364431/137229547-1efcf7bb-5432-4119-85e9-d46b9a3f9ac4.png)
